packages/xo-web/src/xo-app/pool/tab-advanced.js

```diff
@@ -359,6 +362,7 @@ export default class TabAdvanced extends Component {
     const { crashDumpSr } = pool
     const crashDumpSrPredicate = this._getCrashDumpSrPredicate()
     const isEnterprisePlan = getXoaPlan().value >= ENTERPRISE.value
+    const isMigrationCompressionAvailable = pool.migrationCompression === undefined
```

Comment:
If `pool.migrationCompression === undefined`, it means that migration compression is **not** available.
So do something like that instead: `const isMigrationCompressionAvailable = pool.migrationCompression !== undefined`